packages/eas-cli/src/commandUtils/context/contextUtils/getProjectIdAsync.ts

```diff
@@ -79,13 +79,21 @@ export async function getProjectIdAsync(
     const appForProjectId = await AppQuery.byIdAsync(localProjectId);
     if (exp.owner && exp.owner !== appForProjectId.ownerAccount.name) {
       throw new Error(
-        `Project config: Project identified by 'extra.eas.projectId' is not owned by owner specified in the 'owner' field. (project = '${appForProjectId.ownerAccount.name}', config = '${exp.owner}')`
+        `Project config: Project identified by 'extra.eas.projectId' (${
+          appForProjectId.ownerAccount.name
+        }) is not owned by owner specified in the 'owner' field (${exp.owner}). ${learnMore(
+          'https://expo.fyi/eas-project-id'
+        )}`
```

Comment:
We use double quotes to reference paths from app.json:
- https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project/ios/bundleIdentifier.ts#L93
- https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project/android/applicationId.ts

```suggestion
        `Project config: Project identified by "extra.eas.projectId" (${
          appForProjectId.ownerAccount.name
        }) is not owned by owner specified in the "owner" field (${exp.owner}). ${learnMore(
          'https://expo.fyi/eas-project-id'
        )}`
```